### PR TITLE
[LA64_DYNAREC] Added more SSE/SSE2 instructions

### PR DESCRIPTION
**Pull request by factfinding**
* Added F2 0F 11 MOVSD opcode

* Added F2 0F 59 MULSD opcode

* Added F2 0F 5E DIVSD opcode

* Added some log information

### CODE REVIEW

File: src/dynarec/la64/dynarec_la64_f20f.c [hunk 1]
```diff
@@ -62,6 +62,21 @@ uintptr_t dynarec64_F20F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
...
             break;
+        case 0x11:
+            INST_NAME("MOVSD Ex, Gx");
+            nextop = F8;
+            GETG;
+            v0 = sse_get_reg(dyn, ninst, x1, gd, 0);
+            if(MODREG) {
+                ed = (nextop&7)+ (rex.b<<3);
+                d0 = sse_get_reg(dyn, ninst, x1, ed, 0);
+                FMOV_D(d0, v0);
```
Comment (ksco) [inline src/dynarec/la64/dynarec_la64_f20f.c:73]: This is again not correct, the upper 64 bit of the corresponding lsx are left undefined. You should really be careful on the SSE instructions. Use `VEXTRINS_D` instead.

Comment (ksco) [reply src/dynarec/la64/dynarec_la64_f20f.c:73]: (I'll fix this next PR)

Comment (ksco) [reply src/dynarec/la64/dynarec_la64_f20f.c:73]: No, the `vextrins.d` retains the untouched part (and that's what we want here, keep the upper part unchanged), but `fmov.d` make the untouched part undefined. See: https://chipsandcheese.com/2023/02/26/loongsons-lsx-and-lasx-vector-extensions/

Comment (factfinding) [reply src/dynarec/la64/dynarec_la64_f20f.c:73]: I see, thinks a lot!

**Pull request merged**